ui/app/AppMain.qml

```diff
@@ -100,7 +105,209 @@ RowLayout {
         }
     }
 
-    signal settingsLoaded()
+
+    appNavBar: StatusAppNavBar {
+        height: appMain.height
+
+        navBarChatButton: StatusNavBarTabButton {
+            icon.name: "chat"
+            checked: !chatsModel.communities.activeCommunity.active  && appView.currentIndex === Utils.getAppSectionIndex(Constants.chat)
+            tooltip.text: qsTr("Chat")
+            onClicked: {
+                if (chatsModel.communities.activeCommunity.active) {
+                    chatLayoutContainer.chatColumn.input.hideExtendedArea();
+                    chatsModel.communities.activeCommunity.active = false
+                }
+                appMain.changeAppSection(Constants.chat)
+            }
+        }
+
+        navBarCommunityTabButtons.model: chatsModel.communities.joinedCommunities
+        navBarCommunityTabButtons.delegate: StatusNavBarTabButton {
+            onClicked: {
+                if (mouse.button === Qt.RightButton) {
+                    chatsModel.communities.setObservedCommunity(model.id)
+                    communityContextMenu.popup()
+                    return
+                }
+                appMain.changeAppSection(Constants.chat)
+                chatsModel.communities.setActiveCommunity(model.id)
+            }
+
+            anchors.horizontalCenter: parent.horizontalCenter
+
+            checked: chatsModel.communities.activeCommunity.active && chatsModel.communities.activeCommunity.id === model.id
+            name: model.name
+            tooltip.text: model.name
+            icon.color: model.communityColor
+            icon.source: model.thumbnailImage
+
+            badge.visible: model.unviewedMessagesCount > 0
+            badge.value: model.unviewedMessagesCount
+            badge.border.color: hovered ? Theme.palette.statusBadge.hoverBorderColor : Theme.palette.statusBadge.borderColor
+            badge.border.width: 2
+
+            StatusPopupMenu {
+                id: communityContextMenu
+
+                StatusMenuItem {
+                    text: qsTr("Invite People")
+                    icon.name: "share-ios"
+                    enabled: chatsModel.communities.observedCommunity.canManageUsers
+                    onTriggered: openPopup(inviteFriendsToCommunityPopup, { communityId: model.id })
+                }
+
+                StatusMenuItem {
+                    text: qsTr("View Community")
+                    icon.name: "group"
+                    onTriggered: openPopup(communityMembersPopup, {community: chatsModel.communities.observedCommunity})
+                }
+
+                StatusMenuItem {
+                    enabled: false//chatsModel.communities.observedCommunity.admin
```

Comment:
Leftover?

Reply:
Nope, I've just kept the comment in place. See: https://github.com/status-im/status-desktop/pull/2699/files#diff-0067538414dc279244337c270963fba7e96ad02472aa3c18d1007f3ceaa8ce81L57